<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>clockerrorsdialog</class>
 <widget class="QDialog" name="clockerrorsdialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>416</width>
    <height>302</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Adjust SI Unit Times</string>
  </property>
  <widget class="QDialogButtonBox" name="buttonSet">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>270</y>
     <width>341</width>
     <height>32</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
  <widget class="QLineEdit" name="controlEdit">
   <property name="geometry">
    <rect>
     <x>52</x>
     <y>32</y>
     <width>61</width>
     <height>20</height>
    </rect>
   </property>
   <property name="maximumSize">
    <size>
     <width>133</width>
     <height>16777215</height>
    </size>
   </property>
   <property name="inputMethodHints">
    <set>Qt::ImhDigitsOnly</set>
   </property>
   <property name="maxLength">
    <number>3</number>
   </property>
  </widget>
  <widget class="QLineEdit" name="secondsEdit">
   <property name="geometry">
    <rect>
     <x>191</x>
     <y>32</y>
     <width>61</width>
     <height>20</height>
    </rect>
   </property>
   <property name="inputMethodHints">
    <set>Qt::ImhDigitsOnly</set>
   </property>
   <property name="maxLength">
    <number>5</number>
   </property>
  </widget>
  <widget class="QTextBrowser" name="listAdjustments">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>70</y>
     <width>351</width>
     <height>161</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="setButton">
   <property name="geometry">
    <rect>
     <x>323</x>
     <y>31</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Set</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>11</x>
     <y>31</y>
     <width>35</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Control</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>31</y>
     <width>55</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Adjustment</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>31</y>
     <width>47</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>(seconds)</string>
   </property>
  </widget>
  <widget class="QLabel" name="notesLabel">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>240</y>
     <width>351</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
   <property name="wordWrap">
    <bool>true</bool>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonSet</sender>
   <signal>accepted()</signal>
   <receiver>clockerrorsdialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonSet</sender>
   <signal>rejected()</signal>
   <receiver>clockerrorsdialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
